pkg/volume/csi/csi_attacher.go

```diff
@@ -169,6 +169,10 @@ func (c *csiAttacher) waitForVolumeAttachmentInternal(volumeHandle, attachID str
 func (c *csiAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.NodeName) (map[*volume.Spec]bool, error) {
 	klog.V(4).Info(log("probing attachment status for %d volume(s) ", len(specs)))
 
+	if c.plugin.volumeAttachmentLister == nil {
```

Comment:
Any reason why this can't fall back on querying the API server if `c.plugin.volumeAttachmentLister` is nil?

Reply:
Ack